info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T02:46:49.337Z"}
info: Connected to the database. {"timestamp":"2022-11-16T02:46:49.371Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newNotesValidator (E:\express\fundo-notes-project\src\validators\/notes.validator.js:15:20)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at logger (E:\express\fundo-notes-project\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at E:\express\fundo-notes-project\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (E:\express\fundo-notes-project\node_modules\raw-body\index.js:231:16)
    at done (E:\express\fundo-notes-project\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (E:\express\fundo-notes-project\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:525:35)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1358:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-11-16T03:26:36.382Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T03:34:36.306Z"}
info: Connected to the database. {"timestamp":"2022-11-16T03:34:36.328Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T03:38:21.086Z"}
info: Connected to the database. {"timestamp":"2022-11-16T03:38:21.113Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T03:38:50.958Z"}
info: Connected to the database. {"timestamp":"2022-11-16T03:38:50.981Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T03:39:46.096Z"}
info: Connected to the database. {"timestamp":"2022-11-16T03:39:46.123Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T03:57:59.967Z"}
info: Connected to the database. {"timestamp":"2022-11-16T03:57:59.992Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T03:59:36.195Z"}
info: Connected to the database. {"timestamp":"2022-11-16T03:59:36.220Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:00:16.398Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:00:16.422Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:04:15.181Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:04:15.224Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:04:30.637Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:04:30.661Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:06:41.338Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:06:41.388Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:07:44.447Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:07:44.479Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:08:26.212Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:08:26.270Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:25:19.792Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:25:19.819Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:26:39.204Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:26:39.227Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:32:13.521Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:32:13.543Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:32:23.603Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:32:23.629Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:35:00.848Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:35:00.875Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:35:05.204Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:35:05.339Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:37:49.995Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:37:50.018Z"}
error: 
    status - 500 
    message - ReferenceError: data is not defined
    at _callee6$ (E:\express\fundo-notes-project\src\controllers\/notes.controller.js:107:19)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/notes/isArchived/6374681b3b606e07e80ff368 
    method - PUT 
    IP - ::1
   {"timestamp":"2022-11-16T04:38:04.338Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:39:03.825Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:39:03.850Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:41:04.946Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:41:04.972Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:42:08.964Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:42:08.987Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:42:21.705Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:42:21.738Z"}
error: 
    status - 500 
    message - ReferenceError: body is not defined
    at _callee6$ (E:\express\fundo-notes-project\src\services\/notes.service.js:45:9)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.updateisArchivedField (E:\express\fundo-notes-project\src\services\/notes.service.js:40:35)
    at _callee6$ (E:\express\fundo-notes-project\src\controllers\/notes.controller.js:104:41)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at updateisArchivedField (E:\express\fundo-notes-project\src\controllers\/notes.controller.js:102:35)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at _callee$ (E:\express\fundo-notes-project\src\middlewares\/auth.middleware.js:25:5)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/notes/isArchived/6374681b3b606e07e80ff368 
    method - PUT 
    IP - ::1
   {"timestamp":"2022-11-16T04:42:23.487Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:47:46.942Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:47:46.964Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:49:06.644Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:49:06.673Z"}
error: 
    status - 500 
    message - ReferenceError: body is not defined
    at _callee6$ (E:\express\fundo-notes-project\src\services\/notes.service.js:45:9)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.updateisArchivedField (E:\express\fundo-notes-project\src\services\/notes.service.js:40:35)
    at _callee6$ (E:\express\fundo-notes-project\src\controllers\/notes.controller.js:104:41)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at updateisArchivedField (E:\express\fundo-notes-project\src\controllers\/notes.controller.js:102:35)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at _callee$ (E:\express\fundo-notes-project\src\middlewares\/auth.middleware.js:25:5)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/notes/isArchived/6374681b3b606e07e80ff368 
    method - PUT 
    IP - ::1
   {"timestamp":"2022-11-16T04:49:10.184Z"}
error: 
    status - 500 
    message - ReferenceError: body is not defined
    at _callee6$ (E:\express\fundo-notes-project\src\services\/notes.service.js:45:9)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.updateisArchivedField (E:\express\fundo-notes-project\src\services\/notes.service.js:40:35)
    at _callee6$ (E:\express\fundo-notes-project\src\controllers\/notes.controller.js:104:41)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at updateisArchivedField (E:\express\fundo-notes-project\src\controllers\/notes.controller.js:102:35)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at _callee$ (E:\express\fundo-notes-project\src\middlewares\/auth.middleware.js:25:5)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/notes/isArchived/6374681b3b606e07e80ff368 
    method - PUT 
    IP - ::1
   {"timestamp":"2022-11-16T04:49:12.235Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:52:20.522Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:52:20.543Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "{ _id: undefined }" (type Object) at path "_id" for model "Notes"
    at model.Query.exec (E:\express\fundo-notes-project\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (E:\express\fundo-notes-project\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/notes/isArchived/6374681b3b606e07e80ff368 
    method - PUT 
    IP - ::1
   {"timestamp":"2022-11-16T04:52:21.166Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:54:02.368Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:54:02.396Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:55:14.692Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:55:14.717Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:55:25.064Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:55:25.112Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T04:58:50.427Z"}
info: Connected to the database. {"timestamp":"2022-11-16T04:58:50.506Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:00:33.651Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:00:33.675Z"}
error: 
    status - 500 
    message - ReferenceError: update is not defined
    at _callee6$ (E:\express\fundo-notes-project\src\services\/notes.service.js:43:9)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.updateisArchivedField (E:\express\fundo-notes-project\src\services\/notes.service.js:40:35)
    at _callee6$ (E:\express\fundo-notes-project\src\controllers\/notes.controller.js:104:41)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at updateisArchivedField (E:\express\fundo-notes-project\src\controllers\/notes.controller.js:102:35)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at _callee$ (E:\express\fundo-notes-project\src\middlewares\/auth.middleware.js:25:5)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/notes/isArchived/6374681b3b606e07e80ff368 
    method - PUT 
    IP - ::1
   {"timestamp":"2022-11-16T05:00:34.125Z"}
error: 
    status - 500 
    message - ReferenceError: update is not defined
    at _callee6$ (E:\express\fundo-notes-project\src\services\/notes.service.js:43:9)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.updateisArchivedField (E:\express\fundo-notes-project\src\services\/notes.service.js:40:35)
    at _callee6$ (E:\express\fundo-notes-project\src\controllers\/notes.controller.js:104:41)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at updateisArchivedField (E:\express\fundo-notes-project\src\controllers\/notes.controller.js:102:35)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at _callee$ (E:\express\fundo-notes-project\src\middlewares\/auth.middleware.js:25:5)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/notes/isArchived/6374681b3b606e07e80ff368 
    method - PUT 
    IP - ::1
   {"timestamp":"2022-11-16T05:00:35.520Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:01:13.168Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:01:13.195Z"}
error: 
    status - 500 
    message - ReferenceError: isArchived is not defined
    at _callee6$ (E:\express\fundo-notes-project\src\services\/notes.service.js:43:9)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.updateisArchivedField (E:\express\fundo-notes-project\src\services\/notes.service.js:40:35)
    at _callee6$ (E:\express\fundo-notes-project\src\controllers\/notes.controller.js:104:41)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12
    at updateisArchivedField (E:\express\fundo-notes-project\src\controllers\/notes.controller.js:102:35)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at _callee$ (E:\express\fundo-notes-project\src\middlewares\/auth.middleware.js:25:5)
    at tryCatch (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\express\fundo-notes-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/notes/isArchived/6374681b3b606e07e80ff368 
    method - PUT 
    IP - ::1
   {"timestamp":"2022-11-16T05:01:15.831Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:01:51.218Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:01:51.253Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:02:23.608Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:02:23.632Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:02:54.190Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:02:54.213Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:05:14.335Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:05:14.359Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:08:27.540Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:08:27.564Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:14:50.042Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:14:50.068Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:17:05.488Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:17:05.516Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:18:28.806Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:18:28.832Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:18:44.323Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:18:44.345Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:19:15.437Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:19:15.459Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:19:28.219Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:19:28.256Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:20:45.050Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:20:45.074Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:20:57.347Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:20:57.374Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:21:51.683Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:21:51.709Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:22:15.522Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:22:15.554Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newNotesValidator (E:\express\fundo-notes-project\src\validators\/notes.validator.js:15:20)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at logger (E:\express\fundo-notes-project\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at E:\express\fundo-notes-project\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (E:\express\fundo-notes-project\node_modules\raw-body\index.js:231:16)
    at done (E:\express\fundo-notes-project\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (E:\express\fundo-notes-project\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:525:35)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1358:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-11-16T05:22:16.942Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newNotesValidator (E:\express\fundo-notes-project\src\validators\/notes.validator.js:15:20)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at logger (E:\express\fundo-notes-project\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at E:\express\fundo-notes-project\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (E:\express\fundo-notes-project\node_modules\raw-body\index.js:231:16)
    at done (E:\express\fundo-notes-project\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (E:\express\fundo-notes-project\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:525:35)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1358:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-11-16T05:22:18.423Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newNotesValidator (E:\express\fundo-notes-project\src\validators\/notes.validator.js:15:20)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at logger (E:\express\fundo-notes-project\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at E:\express\fundo-notes-project\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (E:\express\fundo-notes-project\node_modules\raw-body\index.js:231:16)
    at done (E:\express\fundo-notes-project\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (E:\express\fundo-notes-project\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:525:35)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1358:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-11-16T05:22:19.580Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newNotesValidator (E:\express\fundo-notes-project\src\validators\/notes.validator.js:15:20)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at logger (E:\express\fundo-notes-project\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at E:\express\fundo-notes-project\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (E:\express\fundo-notes-project\node_modules\raw-body\index.js:231:16)
    at done (E:\express\fundo-notes-project\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (E:\express\fundo-notes-project\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:525:35)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1358:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-11-16T05:22:20.812Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:23:04.401Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:23:04.429Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newNotesValidator (E:\express\fundo-notes-project\src\validators\/notes.validator.js:15:20)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at logger (E:\express\fundo-notes-project\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at E:\express\fundo-notes-project\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (E:\express\fundo-notes-project\node_modules\raw-body\index.js:231:16)
    at done (E:\express\fundo-notes-project\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (E:\express\fundo-notes-project\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:525:35)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1358:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-11-16T05:23:06.269Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newNotesValidator (E:\express\fundo-notes-project\src\validators\/notes.validator.js:15:20)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at logger (E:\express\fundo-notes-project\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at E:\express\fundo-notes-project\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (E:\express\fundo-notes-project\node_modules\raw-body\index.js:231:16)
    at done (E:\express\fundo-notes-project\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (E:\express\fundo-notes-project\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:525:35)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1358:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-11-16T05:23:08.483Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newNotesValidator (E:\express\fundo-notes-project\src\validators\/notes.validator.js:15:20)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at logger (E:\express\fundo-notes-project\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at E:\express\fundo-notes-project\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (E:\express\fundo-notes-project\node_modules\raw-body\index.js:231:16)
    at done (E:\express\fundo-notes-project\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (E:\express\fundo-notes-project\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:525:35)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1358:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-11-16T05:23:16.381Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newNotesValidator (E:\express\fundo-notes-project\src\validators\/notes.validator.js:15:20)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at logger (E:\express\fundo-notes-project\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at E:\express\fundo-notes-project\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (E:\express\fundo-notes-project\node_modules\raw-body\index.js:231:16)
    at done (E:\express\fundo-notes-project\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (E:\express\fundo-notes-project\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:525:35)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1358:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-11-16T05:23:17.947Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newNotesValidator (E:\express\fundo-notes-project\src\validators\/notes.validator.js:15:20)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at logger (E:\express\fundo-notes-project\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at E:\express\fundo-notes-project\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (E:\express\fundo-notes-project\node_modules\raw-body\index.js:231:16)
    at done (E:\express\fundo-notes-project\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (E:\express\fundo-notes-project\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:525:35)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1358:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-11-16T05:23:20.263Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:23:33.813Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:23:33.838Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:24:51.867Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:24:51.891Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:25:01.236Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:25:01.272Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newNotesValidator (E:\express\fundo-notes-project\src\validators\/notes.validator.js:15:20)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at logger (E:\express\fundo-notes-project\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at E:\express\fundo-notes-project\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (E:\express\fundo-notes-project\node_modules\raw-body\index.js:231:16)
    at done (E:\express\fundo-notes-project\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (E:\express\fundo-notes-project\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:525:35)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1358:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-11-16T05:25:04.695Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newNotesValidator (E:\express\fundo-notes-project\src\validators\/notes.validator.js:15:20)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (E:\express\fundo-notes-project\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:175:3)
    at router (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at logger (E:\express\fundo-notes-project\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (E:\express\fundo-notes-project\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:328:13)
    at E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:346:12)
    at next (E:\express\fundo-notes-project\node_modules\express\lib\router\index.js:280:10)
    at E:\express\fundo-notes-project\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (E:\express\fundo-notes-project\node_modules\raw-body\index.js:231:16)
    at done (E:\express\fundo-notes-project\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (E:\express\fundo-notes-project\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:525:35)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1358:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-11-16T05:25:06.069Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:26:15.258Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:26:15.286Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:26:25.542Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:26:25.579Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:26:30.771Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:26:30.798Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:27:37.145Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:27:37.177Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:29:35.027Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:29:35.052Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:29:47.087Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:29:47.117Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:32:50.923Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:32:50.949Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:33:49.039Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:33:49.065Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:34:13.890Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:34:13.924Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:34:28.357Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:34:28.382Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:34:45.864Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:34:45.903Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:37:56.352Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:37:56.374Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:38:10.604Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:38:10.635Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:40:16.726Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:40:16.748Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:40:50.742Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:40:50.768Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:41:08.387Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:41:08.412Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:42:03.570Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:42:03.595Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:42:26.042Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:42:26.066Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:42:32.261Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:42:32.283Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-11-16T05:42:46.020Z"}
info: Connected to the database. {"timestamp":"2022-11-16T05:42:46.054Z"}
